picard/ui/options/plugins.py

```diff
@@ -86,6 +87,8 @@ def __init__(self, parent=None):
         self.ui.plugins.mimeTypes = self.mimeTypes
         self.ui.plugins.dropEvent = self.dropEvent
         self.ui.plugins.dragEnterEvent = self.dragEnterEvent
+        # hide column initially, shows only when update is available
+        self.ui.plugins.hideColumn(COLUMN_UPDATE)
```

Comment:
This column is hidden here, but where/when is it shown ?

Reply:
zas in line no 285 i set the text of header . setting the text make it visible. Ideally i should have move this text setting in ui file and then has to hide it but it doesn't work in that way. Hide colunmn doesn't work if i set header text in ui_options_plugin.py file. I don't know why

Reply:
This hidden column only contains "Upgrade" button when an upgrade is possible, right ?
What about just displaying the button to upgrade in column displaying the version, something like "Upgrade from 0.1 to 0.2" ? 
I think it would be simpler (just display a QButton instead of a QLabel)

@samj1912 : what do you think?

Reply:
@zas This looks good to me also,  one more thing we can do is put a check box for auto update.

Reply:
@zas sounds good to me too.